checkout-fees-for-woocommerce/checkout-fees-for-woocommerce.php

```diff
@@ -91,11 +91,15 @@ public static function instance() {
 		public function __construct() {
 
 			// Set up localisation.
-			load_plugin_textdomain( 'checkout-fees-for-woocommerce', false, dirname( plugin_basename( __FILE__ ) ) . '/langs/' );
+			// load_plugin_textdomain( 'checkout-fees-for-woocommerce', false, dirname( plugin_basename( __FILE__ ) ) . '/langs/' );.
 
 			// Include required files.
 			$this->includes();
 
+			if ( is_admin() ) {
```

Comment:
instead of this snippet, can we directly call the function? like `$this->load_plugin_textdomain()`